prince-of-versions/src/main/java/co/infinum/princeofversions/mvp/presenter/impl/PovPresenterImpl.java

```diff
@@ -92,4 +111,90 @@ public void onError(@ErrorCode int error) {
             }
         });
     }
+
+    private boolean checkIfUpdateShouldBeMade(VersionContext version) {
```

Comment:
I believe I misunderstood you or Dino when we talked about the minSdk problem. Thats why I created this piece of code just to realize it does nothing. :/
My bad, it wont happen again.